mgmtsystem_claim/mgmtsystem_claim.xml

```diff
@@ -7,7 +7,9 @@
 		<field name="name">Management System Claims Tree</field>
 		<field name="model">mgmtsystem.claim</field>
 		<field name="arch" type="xml">
-			<tree string="Claims" colors="blue:state=='pending' and not(date_deadline and (date_deadline &lt; current_date));gray:state in ('close', 'cancel');red:date_deadline and (date_deadline &lt; current_date)">
+			<tree string="Claims" >
```

Comment:
The red colour can be put here: `red:date_deadline and (date_deadline &lt; current_date)`